.travis.yml

```diff
@@ -19,14 +20,16 @@ addons:
       - gcc
       - binutils-dev
       - libiberty-dev
+      - libunwind8-dev
 
 script:
   - cargo build --release --verbose
   - cargo test --release --verbose
   - cargo doc --release --verbose
   - cargo clippy --release --verbose
+  - if [ "$(rustup show | grep default | grep stable)" != "" ]; then cd fuzz && cargo test --verbose && ./travis-fuzz.sh; fi
```

Comment:
I was wondering: If I interpret this correctly it states that we only want to fuzz on stable right?
Is there a reason why we want to fuzz on stable but not others?

Reply:
Yeah, this seems to be checking that the default rustup target is 'stable'. We could likely target multiple versions with the change to GithubActions?